ui/quadicons.md

```diff
@@ -0,0 +1,11 @@
+### Quadicons
+
+Quadicons are a condensed graphical way to represent information about various entities and their relations.
+
+Example: 4 VM quadicons:
+
+Quadicons are presen in Explorer trees, in GTLs in Listnavs at least.
+
+![Textual Summary Example](../images/quadicon.png)
+
+Currently quadicons are rendered using `QuadiconHelper` in `app/helpers/quadicon_helper.rb`.
```

Comment:
Likely want to uppercase **Quadicons** here.  Its a brand thing so ya might want to take it a step further and bold it everywhere...